configs/language.config.js

```diff
@@ -46,7 +46,7 @@ const LANGUAGES_CONFIG = {
         memory: ALLOWED_RAM * ONE_MB,
     },
     [PROMPTV1]: {
-        model: 'gpt-4-1106-preview',
+        model: 'gpt-4o-2024-05-13',
```

Comment:
Why not add a new language config for `gpt-4o` rather than updating PROMPTV1 ? Someone in future might still want to use PROMPTV1 and hence will again come and add `gpt-4` again.